cmd/opm/alpha/template/composite.go

```diff
@@ -36,6 +73,70 @@ and a 'composite template' file`,
 				log.Fatalf("invalid --output value %q, expected (json|yaml)", output)
 			}
 
+			catalogBuilderMap := make(composite.CatalogBuilderMap)
+
+			wd, err := os.Getwd()
+			if err != nil {
+				log.Fatalf("getting current working directory: %v", err)
+			}
+
+			// setup the builders for each catalog
+			setupFailed := false
+			setupErrors := map[string][]string{}
+			for _, catalog := range catalogConfig.Catalogs {
+				errs := []string{}
+				if catalog.Destination.BaseImage == "" {
```

Comment:
IIRC, this field was commented out as part of https://github.com/operator-framework/operator-registry/pull/1090